test/unit/app/test_remote_shell.py

```diff
@@ -42,6 +42,8 @@ def setUpClass(cls):
         cls.cli_interface = CliInterface()
 
     def test_secure_shell_plugin_without_strict(self):
+        if not mockssh:
+            raise unittest.SkipTest("Skipping tests that require mockssh")
```

Comment:
maybe factor out into a helper to get rid of duplicate code? But looks fine either way.